app/factories/bulkrax/object_factory.rb

```diff
@@ -102,7 +103,8 @@ def find_or_create
     end
 
     def search_by_identifier
-      query = { work_identifier =>
+      work_index = ::ActiveFedora.index_field_mapper.solr_name(work_identifier, :facetable)
```

Comment:
1. Will `work_identifier` always be `:facetable`? Is there a chance this will different project-to-project, and thus fail sometimes? 
2. Is there a simple way to achieve this without calling `ActiveFedora` directly? With [Valkyrization](https://github.com/samvera-labs/bulkrax/projects/3) on the horizon, using `ActiveFedora` directly is going the way of the dinosaur. If the answer is "no" that's fine, I don't think it should block this PR; [this ticket](https://github.com/samvera-labs/bulkrax/issues/829) will rework this line later

Reply:
I think the answer with both is to add a new config variable to bulkrax with a default that just is `"#{work_identifier}_ssim"`

Reply:
I'm reverting this change because it's breaking my ability to import FileSet's using the row.

ref: https://github.com/samvera-labs/bulkrax/issues/867